src/cli/ghost-sync.js

```diff
@@ -29,9 +29,9 @@ const writeRevisions = async (revisionsFile, revisions) => {
   })
 }
 
-const writeFile = folderPath => async ({ file, content }) => {
+const writeFile = folderPath => async ({ file, content, deleted }) => {
   const filePath = path.join(folderPath, file)
-  return fs.writeFileAsync(filePath, content)
+  return deleted ? fs.unlink(filePath) : fs.writeFileAsync(filePath, content)
```

Comment:
you want to use `unlinkAsync` which would be the promisified version here. That means the result of the unlink operation will be properly awaited

Reply:
OK, fixed that.